docs/smart-contracts/tokens/wayfinding.mdx

```diff
@@ -0,0 +1,80 @@
+description: Choose between the Stellar Asset Contract and Custom Tokens
+---
+
+Tokens exist in two forms on Stellar:
+
+1. Assets issued by Stellar accounts (G... addresses) and their built-in [Stellar Asset Contract (SAC)](./stellar-asset-contract.mdx) implementation, and
+2. [Custom tokens](./token-interface.mdx) issued by a deployed WASM contract (C... addresses).
+
+Several factors can help you determine whether to issue an asset on Stellar or create a custom token with a smart contract for your project.
+
+However:
+
+### TL;DR
+
+If possible, we recommend issuing a Stellar asset and using the SAC to interact with that asset in smart contracts or to send to contract addresses. More on why below.
+
+## Issuing assets on Stellar
+
+Stellar has first-class support for asset tokenization — issuing an asset can be done using a [built-in transaction](https://developers.stellar.org/docs) without the development of a smart contract.
+
+Stellar’s transactions are fast and cost-effective, making the network great for remittances and micropayments. It also has built-in features for compliance, asset management, and auditing. If you are looking to perform transfers of value, issuing assets on Stellar has all the needed capabilities.
+
+Stellar assets:
+
+- Are compatible with Stellar ecosystem products (such as Stellar wallets) and other ecosystem products (such as exchanges).
+- Benefit from [anchors](../../learn/encyclopedia/anchors.mdx), the bridges between the Stellar network and traditional financial systems. Explore the global [Stellar anchor directory](https://anchors.stellar.org/) for further details.
+- Give the issuer granular control over asset management with features that allow the issuer to [name the asset](../../issuing-assets/control-asset-access.mdx#naming-an-asset), [determine access control](../../issuing-assets/control-asset-access.mdx#controlling-access-to-an-asset-with-flags), [limit asset supply](../../issuing-assets/control-asset-access.mdx#limiting-the-supply-of-an-asset), [publish asset information](../../issuing-assets/publishing-asset-info.mdx), and [ensure compliance](../../issuing-assets/anatomy-of-an-asset.mdx#compliance).
+
+:::note
+
+Note that while these items are also possible with custom smart contract tokens, it is more work to build the token contract rather than using the already-implemented features of Stellar asset tokens.
+
+:::
+
+Assets issued on the Stellar network are accessible to smart contracts with the use of that asset’s Stellar Asset Contract (SAC).
+
+### Stellar Asset Contract
+
+The Stellar Asset Contract (SAC) is compiled into the protocol layer and allows smart contracts to interact with assets issued on Stellar. A single instance of the SAC can be deployed for every Stellar asset by anyone who wants to interact with the asset from a contract. The SAC has access to all account (for XLM) and trustline (for all other assets) balances as well as smart contract token balances.
```

Comment:
Couple of small things, missing 's's for plurals, and trust line is two words. Also, having "balances" after the brackets makes it a little awkward to read because the brackets are inserted into the thing being referenced rather than after it.
```suggestion
The Stellar Asset Contract (SAC) is compiled into the protocol layer and allows smart contracts to interact with assets issued on Stellar. A single instance of the SAC can be deployed for every Stellar asset by anyone who wants to interact with the asset from a contract. The SAC has access to all account balances (for XLM) and trust line balances (for all other assets) as well as smart contract balances.
```